app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -38,6 +41,7 @@
 
     row.serviceBindingsVersion = APIService.getPreferredVersion('servicebindings');
     row.serviceInstancesVersion = APIService.getPreferredVersion('serviceinstances');
+    row.isMobileService = $filter('isMobileService');
```

Comment:
this has to be left on the controller as the `$onChanges` is called from an event which gives it a different scope. So attaching this to row lets me call it with `row.isMobileService` where I would otherwise be out of scope.

Reply:
Isn't row declared at the top also? Should be the same scope

Reply:
That's a good point, and confusing. I did try to change it as you suggested and the isMobileService became undefined. I also note that the above lines are existing code and also bound to the row object, when I tried changing those to var as well, they also went to undefined.

So, I cannot change it to var, I thought I knew why, but now I realise that I do not!